### PR TITLE
Support imepller in egl renderer

### PR DESCRIPTION
**Pull request by JSUYA**
When there is an impeller argument, set egl config to match the impeller.

### CODE REVIEW

**Review by xiaowei-guan — APPROVED** [review]

**Comment by JSUYA** [comment]
Hi @swift-kim please review this

File: flutter/shell/platform/tizen/tizen_renderer_egl.cc [hunk 1]
```diff
@@ -179,12 +166,50 @@ bool TizenRendererEgl::ChooseEGLConfiguration() {
...
-  if (!eglChooseConfig(egl_display_, config_attribs, configs, config_size,
-                       &num_config)) {
-    free(configs);
-    PrintEGLError();
-    FT_LOG(Error) << "No matching configurations found.";
-    return false;
+  if (enable_impeller_) {
+    EGLint impeller_config_attribs[] = {
+        // clang-format off
+      EGL_SURFACE_TYPE,    EGL_WINDOW_BIT,
```
Comment (swift-kim) [inline flutter/shell/platform/tizen/tizen_renderer_egl.cc:172]: nit: Additional indentation
```suggestion
        EGL_SURFACE_TYPE,    EGL_WINDOW_BIT,
```

**Review by swift-kim — APPROVED** [review]
Otherwise looks good to me.

File: flutter/shell/platform/tizen/tizen_renderer_egl.cc (hunk 1 above)
Comment (JSUYA) [reply flutter/shell/platform/tizen/tizen_renderer_egl.cc]: done!

**Pull request merged**